Modified migrator application to initialize the base Operations class…

fsecada01 opened the pull request:
… for SQLite instances. Can be further developed but this provides a good bootstrapping for evaluating testing on local environments. Linted with Black

aachurin committed:
Initial commit

aachurin committed:
# bump

aachurin committed:
# bump

aachurin committed:
Update README.md

aachurin committed:
Update README.md

aachurin committed:
Update README.md

aachurin committed:
# bump

aachurin committed:
Merge branch 'master' of github.com:aachurin/peewee_migrations

aachurin committed:
Update README.md

aachurin committed:
Error handling in watch

aachurin committed:
fix version in README

aachurin committed:
# 0.3.10

Commit:
Code cleaning

Commit:
# small fixes

Commit:
# small fixes

Commit:
improved model import

Commit:
v0.3.16

Commit:
v0.3.16

Commit:
# fix alert messages

Commit:
fix data migration hints

aachurin committed:
Update README.md

Commit:
fix issue 2

aachurin committed:
Update README.md

Commit:
# add enum support

Commit:
Merge branch 'master' of github.com:aachurin/peewee_migrations

fsecada01 committed:
Modified migrator application to initialize the base Operations class for SQLite instances. Can be further developed but this provides a good bootstrapping for evaluating testing on local environments. Linted with Black

Seluj78 commented:
Hi, thanks for the PR ! I don't think you should've linted with black as it modified all the rest of the code whereas this wasn't needed. Other than that, I am all for this !

Commit:
Always set index param in snapshot for complex postgres types

HStoreField, BinaryJSONField and TSVectorField have index turned on by
default by peewee. Now it is always explicitly set.

aachurin committed:
Merge pull request #20 from tinkoffjournal/fix-index-flag-pg-fields

Always set index param in snapshot for complex postgres types

fsecada01 commented:
Ah!  Just saw this.  I can go about doing a fresh pull then resubmitting, if that works @Seluj78

fsecada01 committed:
Merge branch 'aachurin:master' into master

Seluj78 commented:
So, what's next ? :) Sorry I never saw your reply @fsecada01 !
